youtube_dl/extractor/adobepass.py

```diff
@@ -1424,42 +1424,46 @@ def raise_mvpd_required():
                         mvpd_confirm_page, urlh = mvpd_confirm_page_res
                         if '<button class="submit" value="Resume">Resume</button>' in mvpd_confirm_page:
                             post_form(mvpd_confirm_page_res, 'Confirming Login')
-                elif mso_id == 'Verizon':
```

Comment:
Why remove Verizon codes?

Reply:
I removed them, because I had added them after a PR was merged (after this Dish addition), which resulted in a merge conflict. As far as I know, by removing the Verizon codes here, the merge conflict has been removed. My PR does not include MSOs (such as Verizon) that have been included in merged PRs since this PR was created (if that makes sense).

Reply:
Got it. Could you add them back?